src/mscorlib/src/mscorlib.txt

```diff
@@ -2327,6 +2327,7 @@ ArgumentOutOfRange_Week = The Week parameter must be in the range 1 through 5.
 ; begin System.TimeZoneInfo InvalidTimeZoneException's
 ;
 InvalidTimeZone_InvalidRegistryData = The time zone ID '{0}' was found on the local computer, but the registry information was corrupt.
+InvalidTimeZone_InvalidFileData = The time zone ID '{0}' was found on the local computer, but the file information was corrupt.
```

Comment:
you mentioned file information but it is kind of ambiguous what file is corrupted. can we display the file path too